ci/rpm/scan_daos_maldet.sh

```diff
@@ -2,32 +2,25 @@
 
 set -uex
 
-sudo yum -y install \
-  daos{,-{client,server,tests,debuginfo,devel}}-"${DAOS_PKG_VERSION}"
+# Install the DAOS RPMs and then do a malware scan
+# of the resulting system volume.
 
-lmd_src="maldet-current"
-lmd_tarball="maldetect-current.tar.gz"
-rm -rf "/var/tmp/${lmd_src}"
-mkdir -p "/var/tmp/${lmd_src}"
-tar -C "/var/tmp/${lmd_src}" --strip-components=1 -xf "/var/tmp/${lmd_tarball}"
-pushd "/var/tmp/${lmd_src}"
-  sudo ./install.sh
-popd
-sudo /usr/local/sbin/maldet --update-sigs
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# at some point we want to use: shellcheck source=ci/rpm/distro_info.sh
+# shellcheck disable=SC1091
+source "$mydir/distro_info.sh"
 
-fc_conf="/etc/freshclam.conf"
-if ! sudo grep -q 'ScriptedUpdates no' "$fc_conf"; then
-  sudo -E bash -c "echo \"ScriptedUpdates no\" >> \"$fc_conf\""
-fi
-: "${JOB_URL:=${JENKINS_URL}job/clamav_daily_update/}"
-if ! sudo grep -q "$JENKINS_URL" "$fc_conf"; then
-  clam_url="${JOB_URL}lastSuccessfulBuild/artifact/download/clam"
-  sudo -E bash -c "echo \"PrivateMirror ${clam_url}\" >> \"$fc_conf\""
+if command -v dnf; then
+  sudo dnf -y install \
```

Comment:
```suggestion
  sudo dnf install \
```

Reply:
This install could just be done in the dockerfile however couldn't it?

Reply:
It is much more convenient for me to debug in VMs.